_2017_pages/code_of_conduct.md

```diff
@@ -0,0 +1,189 @@
+  the words used), your account of what happened, and any available supporting records
+  (e.g. email, GitHub issue, screenshots, etc.). The online form supports
+  attaching up to 5 files and email and Slack have not limit.
+* Description of the circumstances/context surrounding the incident. Let us know if the
+  incident is ongoing, and/or if this is part of an ongoing pattern of behavior
+* Names and contact info, if possible, of anyone else who witnessed or was involved in
+  this incident. (Did anyone else observe the incident?)
+* Any other relevant information you believe we should have
+
+_The Diversity committee will attempt to gather and write down the above information from
+anyone making a verbal report. Recording the details in writing is exceedingly important
+in order for us to effectively respond to reports. If event staff write down a report
+taken verbally, then the person making the report will be asked to review the written
+report for accuracy._
+
+### Enforcement
+
+At PyMCon, if a participant engages in behavior that violates the Code of Conduct, the
+conference organizers and staff may take any action they deem appropriate.
+
+Potential consequences for violating the NumFOCUS Code of Conduct at PyMCon include:
+
+* Warning the person to cease their behavior and that any further reports will result in
+  sanctions
+* Requiring that the person avoid any interaction with the person they are harassing for
+  the remainder of the event
+* Ending a talk that violates the policy early
+* Not publishing the video or slides of a talk that violated the policy
+* Not allowing a speaker who violated the policy to give (further) talks at the event
+  now or in the future
+* Immediately ending any event volunteer responsibilities and privileges the reported
+  person holds
+* Requiring that the person not volunteer for future events (either indefinitely or for
+  a certain time period)
+* Expelling the person from the event without a refund
+* Banning the person from future events (either indefinitely or for a certain time period)
+* Any other response that the organizing committee deems necessary and appropriate to
+  the situation
+
+### Person(s) Responsible for Resolving Complaints
+
+All reports of breaches of the code of conduct will be investigated and handled by the
+PyMCon Diversity Committee, consulting the NumFOCUS Code of Conduct Enforcement Team when
+necessary.
+
+The current PyMCon Diversity Committee consists of:
+
+* Oriol Abril-Pla, Diversity Chair
+  - oriol.abril.pla@gmail.com
+* Alexandre Andorra, Diversity Chair
+  - andorra.alexandre@gmail.com
+
+### Conflicts of Interest
+
+In the event of any conflict of interest, the committee member will immediately notify
+and recuse themselves if necessary.
+
+If you are concerned about making a report that will be read by any of the above
+individuals, please reach out to [PyMCon's executive committee]() or to one of the members
```

Comment:
not sure what to link here nor if it should be linked here.

Reply:
The numfocus anonymous. Being honest I don't know exactly know how jekylls markdown rendering works so I'm going to merge this PR as is so I don't mess anything up and create an issue to track this.

Reply:
Numfocus one _can_ be anonymous, but it's not necessary.